### PR TITLE
add tables changelog link to tables readme

### PR DESCRIPTION
**Pull request by mario-guerra**

### CODE REVIEW

File: sdk/tables/Azure.Data.Tables/README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Azure Cosmos DB provides a Table API for applications that are written for Azure
 
 The Azure Tables client library can seamlessly target either Azure Table storage or Azure Cosmos DB table service endpoints with no code changes.
 
-[Source code][table_client_src] | [Package (NuGet)][table_client_nuget_package] | [API reference documentation][api_reference] | [Samples][table_client_samples]
+[Source code][table_client_src] | [Package (NuGet)][table_client_nuget_package] | [API reference documentation][api_reference] | [Samples][table_client_samples] | [Change Log][table_change_log]
```
Comment (christothes) [inline sdk/tables/Azure.Data.Tables/README.md:16]: Is this something we are doing to all clients?

Comment (mario-guerra) [reply sdk/tables/Azure.Data.Tables/README.md:16]: Not necessarily, but we could. This was a specific ask from the Cosmos team when David Berry presented his updated .NET tables example webapp to them.

Comment (christothes) [reply sdk/tables/Azure.Data.Tables/README.md:16]: I think we should consider making this standard

**Review by christothes — APPROVED** [review]

**Comment by mario-guerra** [comment]
#ready-to-merge

**Pull request merged**